wizard.py

```
@@ -308,10 +313,28 @@ def setup_git_hooks():
+    config_file = Path("config.yml")
+    config_template = Path("config.yml.template")
+
+    if not config_file.exists():
+        if config_template.exists():
+            import shutil
+            shutil.copy(config_template, config_file)
+            console.print("✅ [green]Created config.yml from template[/green]")
+        else:
+            console.print("⚠️  [yellow]config.yml.template not found, skipping config setup[/yellow]")
+    else:
+        console.print("ℹ️  [blue]config.yml already exists, keeping existing configuration[/blue]")
```

Ruff (0.14.10)

String contains ambiguous `ℹ` (INFORMATION SOURCE). Did you mean `i` (LATIN SMALL LETTER I)?

(RUF001)